salt/metalk8s/beacon/certificates.sls

```diff
@@ -39,15 +38,29 @@ Add beacon for certificates expiration:
 Add beacon for kubeconfig expiration:
   beacon.present:
     - name: metalk8s_watch_kubeconfig_expiry
-    - save: True
     - beacon_module: metalk8s_kubeconfig_info
     - interval: {{ certificates.beacon.interval }}
     - disable_during_state_run: True
     - notify_days: {{ certificates.beacon.notify_days }}
     - files: {{ kubeconfigs | json }}
 {%- endif %}
 
-{%- if not certs and not kubeconfigs %}
+{%- if certs or kubeconfigs %}
+{# We should use `- save: True` in the `beacon.present` states above,
+   instead of writting the configuration file with a `file.managed`,
```

Comment:
```suggestion
   instead of writing the configuration file with a `file.managed`,
```

Reply:
Mêh, build failed... at least this is now fixed...